app/ledger.js

```diff
@@ -286,6 +287,18 @@ if (ipc) {
     if (balanceTimeoutId) clearTimeout(balanceTimeoutId)
     balanceTimeoutId = setTimeout(getBalance, 5 * msecs.second)
   })
+
+  ipc.on(messages.OPEN_LEDGER_TRANSACTION_CSV, (event, viewingIds, csvFilename) => {
+    if (client) {
+      let txCsvText = client._getTransactionCSVText(viewingIds)
+      let txCsvTextDataURI = 'data:text/csv;base64,' + base64Encode(txCsvText)
+
+      const win = electron.BrowserWindow.getFocusedWindow()
+      if (win && win.webContents) {
+        win.webContents.downloadURL(txCsvTextDataURI)
```

Comment:
unfortunately there seems to be no way to specify the filename so it defaults to `download.csv`. but i'm fine with merging this as-is and fixing the filename in a later issue. thanks!